be/src/http/action/transaction_stream_load.cpp

```diff
@@ -436,7 +436,15 @@ Status TransactionStreamLoadAction::_parse_request(HttpRequest* http_req, Stream
     } else if (!http_req->header(HTTP_COMPRESSION).empty()) {
         request.__set_payload_compression_type(http_req->header(HTTP_COMPRESSION));
     }
-
+    if (!http_req->header(HTTP_INSERT_MODE).empty()) {
```

Comment:
I wonder why don't we reuse the same code snippet?

Reply:
just keep the same code style near this code.